addOns/authhelper/src/main/resources/META-INF/jdoconfig.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<jdoconfig xmlns="https://db.apache.org/jdo/xmlns/jdoconfig"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="https://db.apache.org/jdo/xmlns/jdoconfig
+        https://db.apache.org/jdo/xmlns/jdoconfig_3_2.xsd" version="3.2">
+
+    <persistence-manager-factory name="authhelper">
+
+        <property name="javax.jdo.option.PersistenceUnitName" value="authhelper" />
+        <property name="javax.jdo.PersistenceManagerFactoryClass" value="org.datanucleus.api.jdo.JDOPersistenceManagerFactory" />
+
+        <property name="datanucleus.connectionPool.maxPoolSize" value="2" />
```

Comment:
Just curious, how was this value decided?

Reply:
Based on the usage we'll have, we only write to the session when authenticating and read only when creating reports.